contracts/secret-storage/SecretManager.sol

```diff
@@ -484,7 +514,7 @@ contract SecretManager is
     ) internal view returns(address signer) {
         _checkSignValidity(_signTimestamp);
 
-        bytes32 hashStruct = keccak256(abi.encode(ALIVE_TYPEHASH, _storageTimeUsage, _terminatedSecretIds, _signTimestamp));
+        bytes32 hashStruct = keccak256(abi.encode(ALIVE_TYPEHASH, _storageTimeUsage, _signTimestamp));
```

Comment:
why is this list removed from the signature verification?

Reply:
For some strange reason, the signature verification was failing with the `terminatedSecretIds` array. Couldn't figure out the reason, so removed it just to progress with other test cases. But this needs to be added for sure.
Please check this on your side once.

Reply:
`keccak256(abi.encodePacked(_chainIds))` this is from the gateway registration. Can use it?